docs/cvl/v5-changes.md

```diff
@@ -193,6 +194,52 @@ is the main contract being verified, then passing `--contract C` will cause
 method variables to be instantiated in the same way the would have in older
 versions.
 
+Disallow Solidity function calls in quantified expressions
```

Comment:
I might use "calls to contract functions" instead of "Solidity function calls"

Reply:
Changed.